scripts/check-broken-links.mjs

```diff
@@ -25,16 +68,14 @@ const LOGGER_MAP = {
 // See: https://github.com/JustinBeckwith/linkinator?tab=readme-ov-file#linkinatorcheckoptions
 const LINKCHECKER_CONFIG = {
   // Links in this array will not be checked. Will report as SKIPPED.
-  linksToSkip: [
-    'https://www.googletagmanager.com/',
-    'https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js',
-    // process.env.SKIP_IMAGES ? '' : null
-  ],
-  timeout: 5000, // Fail a link if it doesn't resolve within 5s, otherwise linkinator will hang until it resolves.
+  linksToSkip: LINKS_TO_SKIP,
+  timeout: 10000, // Fail a link if it doesn't resolve, otherwise linkinator will hang until it resolves.
   urlRewriteExpressions: [
     // { pattern: '', replacement: '' }
   ],
   // recurse: true, // not recursing through links that are checked because we scan the full known sitemap
+  retryErrors: true,
+  retryErrorsCount: 3,
```

Comment:
This lowers the amount of false positives, but more importantly it ensures that the bulk of pages are checked. Without this, I was finding that a full script run was only scanning about 3000 of 25000 pages; presumably the remaining 22000 didn't load within the timeout.